### PR TITLE
Docs: Add more common test failure documentation

### PR DESCRIPTION
**Pull request by cecille**

### CODE REVIEW

**Review by raju-apple — APPROVED** [review]

**Review by bauerschwan — APPROVED** [review]

**Review by doublemis1 — APPROVED** [review]
LGTM!

File: docs/common_test_failures/TC-ACE-2.1_2.2.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  <tr>
+   <td>Error reading `&lt;cluster>`:`&lt;attribute>` = `&lt;error>`
+   </td>
+  </tr>
+  <tr>
+   <td>asyncio.exceptions.CancelledError
+   </td>
+   <td>no problem reported - test exception
+   </td>
+   <td>Test has timed out. If you have many attributes or a slow network, this may not indicate a true failure. If the test still appeared to be running when the timeout happened, try adjusting the timeout on in the config or command line using the `timeout` parameter (set in seconds).
```
Comment (ccruzagralopes) [inline docs/common_test_failures/TC-ACE-2.1_2.2.md:43]: I believe you meant to have either "or" or "in" in here, not both
"try adjusting the timeout **on in** the config"

**Review by rquidute — APPROVED** [review]

**Review by ccruzagralopes — APPROVED** [review]

File: docs/common_test_failures/TC-ACE-2.1_2.2.md (hunk 1 above)
Comment (raju-apple) [reply docs/common_test_failures/TC-ACE-2.1_2.2.md:43]: Lets fix the minor nit later , want to merge it in so the SVE folks can see this doc on the repo

**Review by fabiowmm — COMMENTED** [review]
LGTM

**Pull request merged**